packages/api/AlvTimeWebApi/Controllers/Admin/TimeEntriesAdminController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AlvTime.Business.TimeRegistration;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System;
+using AlvTimeWebApi.Authentication;
+using System.Linq;
+using AlvTimeWebApi.Utils;
+
+namespace AlvTimeWebApi.Controllers.Admin
+{
+    [Route("api/admin")]
+    [ApiController]
+    [AuthorizeOrakelet]
+    public class TimeEntriesAdminController : Controller
```

Comment:
Kan arve fra ControllerBase istedet for Controller. Controller legger til en del ekstra som ikke trengs.